question2/hash.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+function set(sent) {
+  // 入力値を配列に格納
+  var sent_arr = sent.split(" ");
```

Comment:
変数名は`snake_case`ではなく`camelCase`にしましょう。
`var`じゃなくて`const` `let`どちらかにしましょう。
* 原則`const`
* 再代入がある変数だけ`let`

この辺もESLintが正しく設定されてれば警告出ます。
```suggestion
  const sentArr = sent.split(" ");
```